apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/PooledSession.cs

```diff
@@ -295,6 +351,21 @@ internal Task<Transaction> BeginTransactionAsync(BeginTransactionRequest request
             return RecordSuccessAndExpiredSessions(Client.BeginTransactionAsync(request, callSettings));
         }
 
+        internal TransactionOptions GetTransactionOptions()
```

Comment:
These won't always be complete options - for example, I think read-only transactions usually have a staleness bound. (There may be other things too.)
Not sure how much of a problem this is. We should probably talk about this. Also, could this method be private?